### PR TITLE
fix: overflow of fields in the user management cards

### LINKED ISSUE #7256: Overflow of some fields such as district in the user management page
**Describe the bug**
Fields like Role and District in the user management page are fixed in width, so if a word of length greater than that width comes there it overflows over other elements or goes out of boundary.

**To Reproduce**

1. Go to https://care.ohc.network/
2. Click on 'Users' tab in the left sidebar
3. Scroll down to the cards

**Expected behavior**
The fields if overflowed should wrap the elements in the next line to align themselves or the text itself can be shortened instead of showing the full names. Eg: Thiruvananthapuram can be Thiruvanan.. and hovering over shows a tooltip with full text.

**Screenshots**
image

**Desktop:**

- OS: Windows 11
- Browser Chrome
- Version 122.0.6261.57 (Official Build) (64-bit)

I would like to work on this issue as well.

### PR DESCRIPTION
**Pull request by alanansari**
## Proposed Changes

- Fixes #7256
- Fixed using overflow-hidden and style={{ textOverflow: "ellipsis" }} which hides overflown text and adds ellipses at the end
- Added the text content to the title so on hover the full text gets displayed in a tooltip
- Screenshot:
- image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Common/UserDetailsComponet.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}: {
+  id: string;
+  title: string;
+  value: string;
+}) => (
+  <div className="col-span-1">
+    <UserDetails id={id} title={title}>
+      <div
+        className="overflow-hidden font-semibold"
+        style={{ textOverflow: "ellipsis" }}
```
Comment (rithviknishad) [inline src/Components/Common/UserDetailsComponet.tsx]: Use tailwindcss
```suggestion
```

File: src/Components/Users/ManageUsers.tsx [hunk 2]
```diff
@@ -336,7 +333,11 @@ export default function ManageUsers() {
                   {user.created_by && (
                     <div className="col-span-1">
                       <UserDetails id="created_by" title="Created by">
-                        <div className="break-all font-semibold">
+                        <div
+                          className="overflow-hidden font-semibold"
+                          style={{ textOverflow: "ellipsis" }}
```
Comment (rithviknishad) [inline src/Components/Users/ManageUsers.tsx]: use tailwindcss
```suggestion
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Common/UserDetailsComponet.tsx (hunk 1 above)
Comment (alanansari) [reply src/Components/Common/UserDetailsComponet.tsx]: > Use tailwindcss

Hi, I've made the requested changes please see.

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**